Possible pymatgen version error

## 🐛 Bug

The following tests are currently failing:
```
FAILED deepchem/data/tests/test_json_loader.py::test_json_loader - assert (0,...
FAILED deepchem/feat/tests/test_materials_featurizers.py::TestMaterialFeaturizers::test_sine_coulomb_matrix
```

## To Reproduce

Steps to reproduce the behavior:

Running deepchem's CI.

## Additional context

I think this may be related to a new pymatgen version release, which would require pinning the pymatgen dependency to an earlier version. I'll try to fix this.

--- a/env.common.yml
+++ b/env.common.yml
@@ -20,7 +20,7 @@ dependencies:
     - pillow
     - pubchempy
     - pyGPGO
-    - pymatgen==v2021.1.28
+    - pymatgen==2021.1.28
     - simdna
     - tensorflow==2.3.*
     - tensorflow_probability==0.11.*

--- a/env.common.yml
+++ b/env.common.yml
@@ -20,7 +20,7 @@ dependencies:
     - pillow
     - pubchempy
     - pyGPGO
-    - pymatgen
+    - pymatgen==v2021.1.28
     - simdna
     - tensorflow==2.3.*
     - tensorflow_probability==0.11.*

--- a/env.common.yml
+++ b/env.common.yml
@@ -20,7 +20,7 @@ dependencies:
     - pillow
     - pubchempy
     - pyGPGO
-    - pymatgen==2021.1.28
+    - pymatgen==2020.12.31
     - simdna
     - tensorflow==2.3.*
     - tensorflow_probability==0.11.*

